packages/arb-token-bridge-ui/src/components/TransferPanel/TokenButton.tsx

```diff
@@ -54,17 +53,11 @@ export function TokenButton(): JSX.Element {
       return 'ETH'
     }
 
-    const addressLowercased = selectedToken.address.toLowerCase()
-    const isUSDC = addressLowercased === CommonAddress.Mainnet.USDC
-    const isL2ArbitrumOne = isNetwork(l2.network.id).isArbitrumOne
-
-    // Special case because token symbol for USDC is different on Mainnet and Arbitrum One
-    if (isUSDC && isL2ArbitrumOne) {
-      return isDepositMode ? 'USDC' : 'USDC.e'
-    }
-
-    return selectedToken.symbol
-  }, [selectedToken, isDepositMode, l2])
+    return sanitizeTokenSymbol(selectedToken.symbol, {
+      erc20L1Address: selectedToken.address,
```

Comment:
lost the `toLowerCase()` here and the lower case operation is not moved to the `santizeTokenSymbol` function